Example/SalsaExample.xcodeproj/project.pbxproj

```diff
@@ -327,22 +294,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-SalsaExample/Pods-SalsaExample-frameworks.sh\"\n";
-			showEnvVarsInLog = 0;
-		};
-		62B06F777953ADA2BD673C62 /* [CP] Embed Pods Frameworks */ = {
-			isa = PBXShellScriptBuildPhase;
-			buildActionMask = 2147483647;
-			files = (
-			);
-			inputPaths = (
-			);
-			name = "[CP] Embed Pods Frameworks";
-			outputPaths = (
-			);
-			runOnlyForDeploymentPostprocessing = 0;
-			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-SalsaExampleTests/Pods-SalsaExampleTests-frameworks.sh\"\n";
+			shellScript = "\"${PODS_ROOT}/Target Support Files/Pods-SalsaExample/Pods-SalsaExample-frameworks.sh\"\n";
```

Comment:
What are the script phase changes doing?

Reply:
This seems to be something that changed between Cocoapods 1.5.3 and Cocoapods 1.6.1.
I'm not 100% sure why that got changed.

Reply:
@elliottwilliams upon deeper insepction, `PODS_ROOT` is created by the later version of Cocoapods.
`pod install` creates a User-Defined env variable which points to what is essentially `${SRCROOT}/Pods`